pyaedt/edb_core/primitives.py

```diff
@@ -0,0 +1,92 @@
+"""
+This module .
+"""
+
+from pyaedt import aedt_exception_handler
+from pyaedt.edb_core.layout import EdbLayout
+from pyaedt.edb_core.padstack import EdbPadstacks
+
+class trace(object):
```

Comment:
use CamelCase for class names
```suggestion
class Trace(object):
```